src/components/app-auto-complete/AppAutoComplete.jsx

```diff
@@ -12,38 +12,45 @@ const defaultFilterOptions = (options, state) => {
 
 const AppAutoComplete = ({
   disabled = false,
+  freeSolo = false,
   fieldValue,
   filterOptions = defaultFilterOptions,
+  ListboxProps = { style: { maxHeight: 150 } },
   loading = false,
   onChange,
+  onInputChange,
```

Comment:
u dont need onChange and onInputChange here also freeSolo and disabled it will be passed as ...props, because u just pass it without modifying

Reply:
I use props for TextField inside AutoComplete, not for itself, so when i pass disable, it will disable only input but i can click on autocomplete and see list of options

freeSolo is autocomplete prop not TextField

OnChange set value only if i chose it from list of options
onInputChange set value when i choose it from list or write in textfield, they can be used for different cases, so i decided to leave both(for example in choosing cities or countries, the better way is to chose only from options list)